### PR TITLE
Update plugin description in code issue #33 :

### PR DESCRIPTION
**Pull request by oussamaLaribi**
c -Add homepage URL
 -Fix organization: **we should also change the organization in Administration → Organization details → Edit organization key to fixing build failure org.sonar.api.utils.MessageException: Project not found.**
 -Fix organization URL
 -Add minimal JRE Specification Version

### CODE REVIEW

**Comment by dedece35** [comment]
Hi @oussamaLaribi ,

please correct the failed checks phase.
thank you.

**Comment by oussamaLaribi** [comment]
> Hi @oussamaLaribi ,
> 
> please correct the failed checks phase. thank you.
Hi @dedece35, 
I don't know why the sonar failed maybe I don't have permission from the new organization. Thanks.

**Review by dedece35 — COMMENTED** [review]
why do you have to need to do this modification ?
I corrected the problem of external PR checks, yesterday .... and it's ok now

for me, non need to modify this github action

**Review by dedece35 — CHANGES_REQUESTED** [review]

**Comment by oussamaLaribi** [comment]
> why do you have to need to do this modification ? I corrected the problem of external PR checks, yesterday .... and it's ok now
> 
> for me, non need to modify this github action

Hi @dedece35, yes I saw it, its ok for me too thanks, I have reverted the changes the other modification is related to issue #33

**Comment by dedece35** [comment]
> > why do you have to need to do this modification ? I corrected the problem of external PR checks, yesterday .... and it's ok now
> > for me, non need to modify this github action
> 
> Hi @dedece35, yes I saw it, its ok for me too thanks, I have reverted the changes the other modification is related to issue #33

Hi @oussamaLaribi, ok. sorry, my bad, I saw that it's a revert and not an update :p after writing my comment :(

File: java-plugin/pom.xml [hunk 1]
```diff
@@ -91,6 +91,7 @@
                     <!-- TODO : check how to resolve it (how to force this version in SonarQube installation) -->
                     <!-- <requirePlugins>java:${sonarjava.version}</requirePlugins> -->
                     <basePlugin>java</basePlugin>
+                    <jreMinVersion>${java.version}</jreMinVersion>
```
[the remark this answers is not in the document]
Comment (oussamaLaribi) [reply java-plugin/pom.xml:94]: No, we don't have a problem. It's just a lack of information
Minimal JRE Specification Version: empty
When we build the plugin, mentioned in  #33

File: php-plugin/pom.xml [hunk 2]
```diff
@@ -56,6 +56,7 @@
                     <sonarLintSupported>true</sonarLintSupported>
                     <sonarQubeMinVersion>${sonarqube.version}</sonarQubeMinVersion>
                     <basePlugin>php</basePlugin>
+                    <jreMinVersion>${java.version}</jreMinVersion>
```
[the remark this answers is not in the document]
Comment (oussamaLaribi) [reply php-plugin/pom.xml:59]: same as the java part also

File: python-plugin/pom.xml [hunk 3]
```diff
@@ -64,6 +64,7 @@
                     <!-- TODO : check how to resolve it (how to force this version in SonarQube installation) -->
                     <!-- <requirePlugins>python:${sonar.python.version}</requirePlugins> -->
                     <basePlugin>python</basePlugin>
+                    <jreMinVersion>${java.version}</jreMinVersion>
```
[the remark this answers is not in the document]
Comment (oussamaLaribi) [reply python-plugin/pom.xml:67]: Same as the java part also

**Comment by oussamaLaribi** [comment]
> > > why do you have to need to do this modification ? I corrected the problem of external PR checks, yesterday .... and it's ok now
> > > for me, non need to modify this github action
> > 
> > 
> > Hi @dedece35, yes I saw it, its ok for me too thanks, I have reverted the changes the other modification is related to issue #33
> 
> Hi @oussamaLaribi, ok. sorry, my bad, I saw that it's a revert and not an update :p after writing my comment :(

Alright, thanks anyway.

File: java-plugin/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dedece35) [reply java-plugin/pom.xml:94]: ok

File: php-plugin/pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dedece35) [reply php-plugin/pom.xml:59]: ok

File: python-plugin/pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dedece35) [reply python-plugin/pom.xml:67]: ok

**Review by dedece35 — APPROVED** [review]

**Comment by dedece35** [comment]
Hi @oussamaLaribi,
PR approved ! 
you can merge with squash option and small comment, please.

**Pull request merged**